Permission audit logs (#7982)
fishtown-analytics/docs.getdbt.com: website/docs/docs/cloud/manage-access/enterprise-permissions.md

</Expandable>
<Expandable alt_header="Stakeholder and Read-Only">

The Stakeholder and Read-Only are identical permission sets that are similar to Viewer, but without access to sensitive content such as account settings, billing information, or audit logs. Useful for personas who need to monitor projects and their configurations.

Notable features: 
- Stakeholder is a project-level set.
- Read-only access to projects, environments, jobs, and runs.
- Read-only access to user and group information.
- Can access <Constant name="explorer" />.
- No access to the IDE.

> - No access to certain sensitive audit log content (such as user settings and account-level changes).

Reviewer: Suggest referring to this as "limited access"
```suggestion
- Limited access to audit log content that excludes sensitive information, such as user settings and account-level changes.
```

Notable features:
- Viewer is an account-level set.
- Read-only access to all settings, projects, environments, and runs.

> - **Read-only access to audit logs** including account-level information (unlike Stakeholder/Read-Only).

Reviewer: That last part might raise more questions than it answers
```suggestion
- Read-only access to audit logs, including sensitive account-level information.
```
